modules/appeals_api/app/models/concerns/appeals_api/higher_level_reviews/formatter.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module HigherLevelReviews
+    class Formatter
+      def initialize(higher_level_review)
+        @higher_level_review = higher_level_review
+      end
+
+      def birth_date
+        AppealsApi::HigherLevelReview::Date.new(
+          higher_level_review.auth_headers.dig('X-VA-Birth-Date')
```

Comment:
I don't really like digging this deep into the data of the HLR to extract relevant data... To me it seems like _where_ the data sits inside the model is the models responsibility and _how_ the data is formatted can be another objects responsibility...